src/d3d12/visualizer.rs

```diff
@@ -140,6 +140,14 @@ impl AllocatorVisualizer {
                             ));
                             ui.text(format!("total block size: {} KiB", total_block_size / 1024));
                             ui.text(format!("total allocated:  {} KiB", total_allocated / 1024));
+                            ui.text(format!(
+                                "num committed resource allocations: {}",
+                                mem_type.committed_allocations.num_allocations
+                            ));
+                            ui.text(format!(
+                                "total committed resource allocations size: {} KiB",
```

Comment:
```suggestion
                                "total committed resource allocation size: {} KiB",
```

(or remove `size`, that's pretty clear from `KiB`?)